rules/postprocess.smk

```diff
@@ -140,60 +131,50 @@ if config["foresight"] != "perfect":
             plotting=config_provider("plotting"),
             heat_sources=config_provider("sector", "heat_pump_sources"),
         input:
-            regions=resources("regions_onshore_base_s_{clusters}.geojson"),
+            regions=resources("regions_onshore.geojson"),
             heat_source_temperature=lambda w: (
-                resources(
-                    "temp_" + w.carrier + "_base_s_{clusters}_temporal_aggregate.nc"
-                )
+                resources("temp_" + w.carrier + "_temporal_aggregate.nc")
                 if w.carrier in ["river_water", "sea_water", "ambient_air"]
                 else []
             ),
             heat_source_energy=lambda w: (
-                resources(
-                    "heat_source_energy_"
-                    + w.carrier
-                    + "_base_s_{clusters}_temporal_aggregate.nc"
-                )
+                resources("heat_source_energy_" + w.carrier + "_temporal_aggregate.nc")
                 if w.carrier in ["river_water"]
                 else []
             ),
         output:
             temp_map=RESULTS
-            + "maps/base_s_{clusters}_{opts}_{sector_opts}_{planning_horizons}-heat_source_temperature_map_{carrier}.html",
-            energy_map=RESULTS
-            + "maps/base_s_{clusters}_{opts}_{sector_opts}_{planning_horizons}-heat_source_energy_map_{carrier}.html",
+            + "maps/{horizon}-heat_source_temperature_map_{carrier}.html",
+            energy_map=RESULTS + "maps/{horizon}-heat_source_energy_map_{carrier}.html",
         threads: 1
         resources:
             mem_mb=150000,
         log:
-            RESULTS
-            + "logs/plot_heat_source_map/base_s_{clusters}_{opts}_{sector_opts}_{planning_horizons}_{carrier}.log",
+            RESULTS + "logs/plot_heat_source_map/{horizon}_{carrier}.log",
         benchmark:
-            (
-                RESULTS
-                + "benchmarks/plot_heat_source_map/base_s_{clusters}_{opts}_{sector_opts}_{planning_horizons}_{carrier}"
-            )
+            (RESULTS + "benchmarks/plot_heat_source_map/{horizon}_{carrier}")
+        conda:
+            "../envs/environment.yaml"
         script:
             "../scripts/plot_heat_source_map.py"
 
 
-if config["foresight"] == "perfect":
+if config["foresight"] == "perfect" and config.get("enable", {}).get(
```

Comment:
No need for `.get()`. Due to assuming `config.default.yaml` is complete, we don't need to handle fallbacks here.